decidim-sortitions/app/controllers/decidim/sortitions/widgets_controller.rb

```diff
@@ -6,15 +6,25 @@ class WidgetsController < Decidim::WidgetsController
       helper Decidim::SanitizeHelper
       helper Sortitions::SortitionsHelper
 
+      def show
+        enforce_permission_to :embed, :sortition, sortition: model if model
+
+        super
+      end
+
       private
 
       def model
-        @model ||= Sortition.where(component: params[:component_id]).find(params[:sortition_id])
+        @model ||= Sortition.where(component: current_component).find(params[:sortition_id])
```

Comment:
Not an actionable here, but some awareness. 
While browsing the sortitions i noticed there is no indication that an embedded sortition is canceled or not.

Reply:
This is out of the scope of the current PR 😄 
I'd guess that's the case because this module isn't used extensively. Also this particular feature (embeds) its not well know and you need to have some knowledge on CMS/coding.